docs/developer/unified-repositories.rst

```diff
@@ -0,0 +1,42 @@
+Unified repository sourcing
+===========================
+
+The Unified feature is used automatically when Anaconda is loading installation repositories for the first time. This feature is used only if package (DNF) installation is active and the remote repository source is configured correctly. It was implemented to simplify the installation experience on systems where multiple repository sources are required to make a successful installation e.g. RHEL, CentOS Stream.
+
+Prerequisites
+-------------
+
+* Package based installation (DNF)
+* URL installation source with the .treeinfo (or treeinfo) file
+
+How this works
+--------------
+
+The Unified feature is used out of the box if the URL you want to use is pointing to a directory with ``.treeinfo`` or ``treeinfo`` file. This file contains metadata about the repository and it might also contain information about other repositories which should be used (the interesting part). The ``.treeinfo`` file may look like
+::
+
+    [variant-AppStream]
```

Comment:
I tried to use this example file (with correct paths to the repositories and without any additional sections), but anaconda is failing to download `remomd.xml`. So, I think the example needs to be extended so that it could be easily used without having to figure out what additional sections are missing.